pkgs/development/libraries/nss/default.nix

```diff
@@ -33,6 +33,10 @@ in stdenv.mkDerivation rec {
 
   patchFlags = "-p0";
 
+  postPatch = stdenv.lib.optionalString stdenv.isDarwin ''
+    substituteInPlace nss/coreconf/Darwin.mk --replace '@executable_path/$(notdir $@)' "$out/lib/\$(notdir \$@)"
```

Comment:
does this work correctly if you need to fixup the install_name afterwards anyway?

Reply:
Yes. It fixes a part of the paths. I couldn’t fix them all in the build. So I ended up hacking the rest with install_name_tool.

Reply:
ah, in that case take a look at [`fixDarwinDylibNames`](https://search.nix.gsc.io/?q=fixDarwinDylibNames&i=nope&files=&repos=), that should handle the fixup for you.

Reply:
Thanks. Didn't notice that when grepping around.